src/helpers/_links.scss

```diff
@@ -44,6 +44,12 @@
   &:active {
     color: $govuk-link-active-colour;
   }
+
+  // When focussed, the text colour needs to be darker to ensure that colour
+  // contrast is still acceptable
+  &:focus {
```

Comment:
I tried changing the focusable mixin and that led to ordering problems since focus was not the last selector.

Reply:
Do you think this would be easy to fix?

Reply:
It just felt fragile having to move lots of code around to get the ordering right, we can explore it more, it would be nice to have it in the mixin.

I think one to do together if we decide to explore that.